external/storm-hdfs/src/main/java/org/apache/storm/hdfs/bolt/AvroGenericRecordBolt.java

```diff
@@ -17,8 +17,9 @@
  */
 package org.apache.storm.hdfs.bolt;
 
+import org.apache.avro.generic.GenericData;
 import org.apache.storm.Config;
-import org.apache.storm.Constants;
+import org.apache.storm.hdfs.avro.AbstractAvroSerializer;
```

Comment:
The deletes look OK but these new imports are not used, can we remove them or revert the file?